src/wpt/streams-test.ts

```diff
@@ -781,36 +712,9 @@ export default {
       'TransformStream: write in detached realm should succeed',
     ],
   },
-  'transform-streams/lipfuzz.any.js': {
-    comment: "I'm not quite sure what this test does",
-    expectedFailures: [
-      'testing "" (length 1)',
-      'testing "" (length 0)',
-      'testing "{{in1}}" (length 1)',
-      'testing "z{{in1}}" (length 1)',
-      'testing "{{in1}}q" (length 1)',
-      'testing "{{in1}}{{in1}" (length 1)',
-      'testing "{{in1}}{{in1},}" (length 2)',
-      'testing "{{in1,}}" (length 2)',
-      'testing "{{,in1}}" (length 2)',
-      'testing "{,{in1}}" (length 2)',
-      'testing "{{,in1}" (length 2)',
-      'testing "{" (length 1)',
-      'testing "{," (length 2)',
-      'testing "{,{,i,n,1,},}" (length 7)',
-      'testing "{{in1}}{{in2}}{{in1}}" (length 1)',
-      'testing "{{wrong}}" (length 1)',
-      'testing "{{wron,g}}" (length 2)',
-      'testing "{{quine}}" (length 1)',
-      'testing "{{bogusPartial}}" (length 1)',
-      'testing "{{bogusPartial}}}" (length 1)',
-    ],
-  },
+  'transform-streams/lipfuzz.any.js': {},
   'transform-streams/patched-global.any.js': {
-    comment: 'To be investigated',
-    expectedFailures: [
-      'TransformStream should use the original value of ReadableStream and WritableStream',
-    ],
+    global: true,
```

Comment:
Can we choose a little bit more verbose key rather than global? It's not really explanatory.